### PR TITLE
[bitnami/airflow] Remove nil values

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

Removes `nil` values from `bitnami/airflow/values.yaml`.

**Checklist** 

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/airflow/values.yaml [hunk 1]
```diff
@@ -148,7 +148,7 @@ serviceAccount:
 ##       - name: portname
 ##         containerPort: 1234
 ##
-sidecars:
+sidecars: []
```
Comment (carrodher) [inline bitnami/airflow/values.yaml:151]:
```suggestion
sidecars: {}
```

File: bitnami/airflow/values.yaml [hunk 2]
```diff
@@ -350,7 +350,7 @@ web:
   ##       - name: portname
   ##         containerPort: 1234
   ##
-  initContainers:
+  initContainers: []
```
Comment (carrodher) [inline bitnami/airflow/values.yaml:353]:
```suggestion
  initContainers: {}
```

**Review by carrodher — COMMENTED** [review]
By agreement in the template used for new charts, we are using `{}` instead of `[]`. See https://github.com/bitnami/charts/blob/master/template/CHART_NAME/values.yaml

**Comment by migruiz4** [comment]
Hi @carrodher,

In that case, I think it is something we should fix in the template.

In the example, you can see initContainers is an "Array of objects", but not an object itself.
```yaml
  ## initContainers:
  ##  - name: your-image-name
  ##    image: your-image
  ##    imagePullPolicy: Always
  ##    command: ['sh', '-c', 'echo "hello world"']
```

**Review by carrodher — COMMENTED** [review]
Can you bump the chart version in the Chart.yaml file? In the same way, the README table should be regenerated by running the automated tool.

And just in case, can you double-check there are no single quotes? If yes, please, replace them from `''` to `""`. The current changes LGTM

**Comment by carrodher** [comment]
There are some `''`:
```console
$ ag ": ''" airflow/values.yaml
97:  key: ''
98:  type: ''
112:podAffinityPreset: ''
358:  priorityClassName: ''
507:  priorityClassName: ''
705:  priorityClassName: ''
852:  bindpw: ''
865:    CAcertificateSecret: ''
869:    CAcertificateFilename: ''
```

**Review by carrodher — APPROVED** [review]
LGTM

**Pull request merged**